spec/fundamentals.mediawiki

```diff
@@ -7,15 +7,37 @@ peer-to-peer trades with the mechanics of order execution existing entirely on
 the separate blockchains via atomic swaps.
 These are:
 * Asset-specific order quantity increments and transaction fee rates
-* Epoch-based pseudorandom order matching
+* Epoch-based pseudorandom order matching, with epoch duration configure per-market
```

Comment:
Is **configure** supposed to be **configured**? I'm also not sure that this detail is necessary here.